src/genqtclist.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Tlf - contest logging program for amateur radio operators
+ * Copyright (C) 2013           Ervin Hegedüs - HA2OS <airween@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+#ifndef GENQTCLIST_H
+#define GENQTCLIST_H
+
+int genqtclist(char * callsign, int nrofqtc);
+void genqtcline(char * qtc, char * line);
```

Comment:
That function is only used in genqtclist() and not needed on other side. The prototype should be moved to the *.c file.